package.json

```diff
@@ -2,8 +2,18 @@
   "name": "eslint-plugin-jest-playwright",
   "description": "ESLint globals for your Jest Playwright setup.",
   "version": "0.2.1",
-  "main": "index.js",
+  "main": "lib/index.js",
   "repository": "https://github.com/mxschmitt/eslint-plugin-jest-playwright",
   "author": "Max Schmitt <max@schmitt.mx>",
-  "license": "MIT"
+  "license": "MIT",
+  "files": [
+    "lib"
+  ],
+  "scripts": {
+    "test": "jest"
```

Comment:
Not sure your preference, but might be nice to add a GitHub action to run these tests on builds?

Reply:
that would be super cool!

Reply:
Sounds good, I'll set one up